README.md

```diff
@@ -178,6 +178,62 @@ applications:
   - conjur
 ```
 
+In PCF version 2.0+, when the service broker creates the Host identity for your application
+in Conjur, it will automatically add it to a Conjur Layer representing the `Organization`
+and `Space` the application is deployed into on PCF. This allows `Spaces` to be privileged
+and any apps deployed into that space to inherit those privileges.
+
+To accomplish this the service broker requires that the Org and Space policy exist prior
+to binding applications with Conjur. The policy to define an organizaiton and space is:
+```yaml
+---
+# Policy for the Organization
+- !policy
+  # Organization GUID from PCF.
+  # This may be obtained by running `cf org --guid {org name}
+  id: cbd7a05a-b304-42a9-8f66-6827ae6f78a1
+  body:
+    # Layer to allow privileging an entire organzation to a resource
+    - !layer
+
+    # Policy for the Spaceß
+    - !policy
+      # Space GUID from PCF.
+      # This may be obtained by running `cf space --guid {spaceß name}
+      id: 8bf39f4a-ebde-437b-9c38-3d234b80631a
+      body:
+        # Layer to allow privileging an entire space to a resource
+        # The service broker will add applications to this layer automatically.
+        - !layer
+
+    # Grant to add the Space layer to the Org Layer
+    - !grant
+      role: !layer
+      member: !layer 8bf39f4a-ebde-437b-9c38-3d234b80631a
+
+# Entitlements to allow the Service broker to manage these policy and layer resources
+- !permit
+  # Organization GUID from PCF
+  resource: !policy cbd7a05a-b304-42a9-8f66-6827ae6f78a1
+  privileges: [ create, update, read ]
+  # This should be the service broker identity you created with configuring the service broker
+  roles: !host service-broker
+
+- !permit
+  # Organization GUID/Space GUID from PCF
+  resource: !policy cbd7a05a-b304-42a9-8f66-6827ae6f78a1/8bf39f4a-ebde-437b-9c38-3d234b80631a
+  privileges: [ create, update, read ]
+  roles: !host service-broker
+
+- !permit
```

Comment:
is this block a duplicate of the block above?

Reply:
It's not, the one above is for the space `!policy`, this one is for the `!layer`.

As I'm thinking about it now, just the one for `!policy` may be sufficient. I will check that now.

Reply:
No, I confirmed we need both the policy and layer permissions for the service broker. We are not able to check that the layer exists unless we have `read` access to it. We don't have that just from the policy permissions.